lib/cts.rb

```diff
@@ -120,7 +120,7 @@ def dacu_disclosure_approver
         if dacu_disclosure_team.approvers.blank?
           raise 'DACU Disclosure team has no approvers assigned.'
         else
-          dacu_disclosure_team.approvers.first
+          dacu_disclosure_team.approvers.last
         end
```

Comment:
why last instead of first?

Reply:
The first user has multiple roles so the tests fail.